src/Libm.jl

```diff
@@ -1,5 +1,17 @@
 module Libm
 
-# package code goes here
+include("erf.jl")
 
-end # module
+
+# Utils
+
+# Get the more significant 32 bit int from a double
+function GET_HIGH_WORD(d::Float64)
```

Comment:
This is perhaps also a good candidate as a utility function to add in a different file.